packages/lab/src/dialog/DialogTitle.css

```diff
@@ -10,7 +10,7 @@
 
 /* Styles applied to the status indicator icon overriding its default size */
 .saltDialogTitle .saltStatusIndicator.saltIcon {
-  --icon-size: var(--salt-text-h2-lineHeight);
+  --saltIcon-size: var(--salt-text-h2-lineHeight);
```

Comment:
I noticed this was using icons variable instead of API as standard when redesigning one of the data grid stories

Reply:
fix this in a separate PR?